lib/ansible/playbook/handler.py

```diff
@@ -38,6 +40,19 @@ def __repr__(self):
         ''' returns a human readable representation of the handler '''
         return "HANDLER: %s" % self.get_name()
 
+    def _validate_listen(self, attr, name, value):
```

Comment:
Instead, I could template listeners in `lib/ansible/plugins/strategy/__init__.py` although only static values have worked in the past. @sivel Do you have an opinion on this?

Reply:
I don't really have an opinion. If we only supported static in the past, I'd say it's fine to continue doing so.  We can always add it as a feature request later if needed.

Reply:
you should not need specific function now, just trigger validation for it

Reply:
^ to explain, _post_validate should already do this for the field, you should not need to create new function that does the same as the generic